### DISCUSSION TITLE
MoveJ or MoveL Commands via ROS2 driver

### DISCUSSION THREAD

**Discussion by pk-robom** [UniversalRobots/Universal_Robots_ROS2_Driver#741] [Q&A] [answered]
Maybe a stupid question, but is it possible to execute regular UR script commands (e.g., MoveJ or MoveL) using the ROS2 driver?
I previously experimented with  [this third-party library](https://gitlab.com/sdurobotics/ur_rtde), which allowed me to do something like this, but would like to use ROS2 as a framework for controlling the robot instead..

**Comment by fmauch** [comment, marked as answer]
Sending custom script commands will be possible soon with  #721
